README.md

```diff
@@ -18,37 +18,36 @@ A Workflow can be packaged as a deployable unit and consumed from any of these i
 
 For a more detailed view of how we think about Lyra, check out our introductory [document](https://docs.google.com/document/d/1oJwg4LlolC3qlt0xG__xjrz16aYwEyOk8GqyNt5_Gdo/edit?usp=sharing)!
 
-
 ## Getting Started
 
 ### Build
-The project requires [Go](https://golang.org/doc/install) 1.11 or higher, and [go modules](https://github.com/golang/go/wiki/Modules) to be on.
+The project requires [Go](https://golang.org/doc/install) 1.11 or higher, and [go modules](https://github.com/golang/go/wiki/Modules) enabled. These instructions should work on Linux and macOS. Proper packages are [coming soon](https://github.com/lyraproj/lyra/issues/12).
 
-1. Clone the git repo: `$ git clone https://github.com/lyraproj/lyra`
-2. Build the binary: `$ cd lyra; make`
+1. Clone this repository: `git clone https://github.com/lyraproj/lyra`
+2. Build Lyra: `cd lyra; make`
 
-### Deploying Workflows with CLI
+### Deploying Workflows with the Lyra CLI
 
 > **!! WARNING: THIS WORKFLOW CREATES REAL RESOURCES ($$) !!**
 
-1. Run the binary with the [sample Workflow](plugins/aws_vpc_yaml.yaml): ` $ ./build/lyra apply aws_vpc_yaml --debug`
-2. Delete the Workflow (i.e. its resources), run ` $ ./build/lyra delete aws_vpc_yaml --debug`.  
+1. Apply a [sample Workflow](workflows/aws_vpc_yaml.yaml): `./build/lyra apply aws_vpc_yaml --debug`
+2. Delete the Workflow (i.e. its resources): `./build/lyra delete aws_vpc_yaml --debug`
 
-This workflow is an AWS Workflow called `aws_vpc_yaml` in `plugins\aws_vpc_yaml.yaml`.  Tag data (loaded [here](plugins/aws_vpc_yaml.yaml#L6) by a new golang implementation of [hiera](https://github.com/lyraproj/hiera)) is specified in the [the data.yaml file](data.yaml) file.  This workflow will use the default AWS credentials configured in your `~/.aws/credentials`.
+This Workflow manages several resources on AWS and incorporates external data (for Tags, loaded [here](workflows/aws_vpc_yaml.yaml#L6) using a new Golang implementation of [hiera](https://github.com/lyraproj/hiera)), as specified in the [data.yaml](data.yaml) file. Lyra will use AWS credentials as configured in `~/.aws/credentials`.
 
-For the examples using Terraform providers (e.g. `typespace=>'TerraformAws'`), region is currently hard-coded to `eu-west-1`. For non-Terraform providers (e.g. `typespace=>'aws'`), Lyra will use the default region supplied in your `~/.aws/config`. 
+> NB regarding regions: For the [examples](workflows/) using Terraform providers (e.g. `typespace=>'TerraformAws'`), region is currently hard-coded to `eu-west-1`. For non-Terraform providers (e.g. `typespace=>'aws'`), Lyra will use the default region supplied in your `~/.aws/config`.
 
 ### Deploying Workflows with Kubernetes
```

Comment:
TODO: Add something about *why* you'd want to do this, how the behaviour is different, etc.